src/textfont.js

```diff
@@ -204,6 +217,13 @@ export class TextFont {
     if (!this.maxSizeGlyph) {
       this.maxSizeGlyph = 'H';
     }
+    this.weight = typeof this.weight === 'undefined' ? '' : this.weight;
+    this.style = typeof this.style === 'undefined' ? '' : this.style;
+    this.setHashBase();
+  }
+
+  setHashBase() {
```

Comment:
This method name reads like a setter. Maybe `initHashBase`?

Reply:
I called it 'update'...